pallets/automation-time/src/lib.rs

```diff
@@ -428,68 +440,6 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Schedule a task through XCMP through proxy account to fire an XCMP message with a provided call.
-		///
-		/// Before the task can be scheduled the task must past validation checks.
-		/// * The transaction is signed
-		/// * The times are valid
-		/// * The given asset location is supported
-		///
-		/// # Parameters
-		/// * `schedule`: The triggering rules for recurring task or the list of unix standard times in seconds for when the task should run.
-		/// * `destination`: Destination the XCMP call will be sent to.
-		/// * `schedule_fee`: The payment asset location required for scheduling automation task.
-		/// * `execution_fee`: The fee will be paid for XCMP execution.
-		/// * `encoded_call`: Call that will be sent via XCMP to the parachain id provided.
-		/// * `encoded_call_weight`: Required weight at most the provided call will take.
-		/// * `overall_weight`: The overall weight in which fees will be paid for XCM instructions.
-		///
-		/// # Errors
-		/// * `InvalidTime`: Time in seconds must be a multiple of SlotSizeSeconds.
-		/// * `PastTime`: Time must be in the future.
-		/// * `DuplicateTask`: There can be no duplicate tasks.
-		/// * `TimeTooFarOut`: Execution time or frequency are past the max time horizon.
-		/// * `TimeSlotFull`: Time slot is full. No more tasks can be scheduled for this time.
-		/// * `Other("proxy error: expected `ProxyType::Any`")`: schedule_as must be a proxy account of type "any" for the caller.
-		#[pallet::call_index(3)]
-		#[pallet::weight(<T as Config>::WeightInfo::schedule_xcmp_task_full(schedule.number_of_executions()).saturating_add(T::DbWeight::get().reads(1)))]
-		pub fn schedule_xcmp_task_through_proxy(
```

Comment:
Can you add comment to your PR? Is this function deprecated?

Reply:
Yes. This function is deprecated.